Sources/SKCore/FileBuildSettings.swift

```diff
@@ -25,9 +25,13 @@ public struct FileBuildSettings: Equatable {
   /// The working directory to resolve any relative paths in `compilerArguments`.
   public var workingDirectory: String? = nil
 
-  public init(compilerArguments: [String], workingDirectory: String? = nil) {
+  /// The isFallback to know if the fileBuildSetting is specified or the default one.
```

Comment:
I would phrase this slightly differently.

```suggestion
  /// Whether the build settings were computed from a real build system or whether they are synthesized fallback arguments while the build system is still busy computing build settings.
```